unittest/bindings/test_shooting.py

```diff
@@ -41,8 +41,8 @@ def test_calc(self):
 
     def test_calcDiff(self):
         # Running calc functions
-        cost = self.PROBLEM.calcDiff(self.xs, self.us)
-        costDer = self.PROBLEM_DER.calcDiff(self.xs, self.us)
+        cost = self.PROBLEM.calcDiff(self.xs, self.us, True)
```

Comment:
Remove `recalc` option. Let's follow the same patter for the shooting problem too.
Call `calc` first

Reply:
Let's discuss this part separately. There is a faulty logic in the defintions of the derived classes in python.

Reply:
Are you suggesting to do not remove `recalc` for shooting problem?

Reply:
Already removed it. There is no recalc anymore.